web/satellite/tests/unit/common/__snapshots__/SearchComponent.spec.ts.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is this file expected to be in here?

Reply:
this file was generated automatically during testing. Should i remove this line?

Reply:
IMO, I don't think we need to have source control for snapshots, but maybe we can see what others think?

Reply:
We need this snapshots to be able to compare that our markup has been changed.

The Jest [docs](https://jestjs.io/docs/en/snapshot-testing.html) also say that "the snapshot artifact should be committed alongside code changes"

Also, this means that the tests won't have a baseline for comparison when run in a fresh environment or during CI

Reply:
Oh, I didn't know we are going to set up running the tests on CI now. That makes sense!

Reply:
@VinozzZ 
Yeah, we are going to make some frontend improvements at the beginnig of september. Adding linter/unit test/auto tests (yes, we have them :D ) to our CI also part of it